### PR TITLE
app-misc/bgrep: bumping to new version

### PR DESCRIPTION
**Pull request by scardracs**
Bumping to bgrep-1_p20210321

Package-Manager: Portage-3.0.17, Repoman-3.0.2
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

File: app-misc/bgrep/bgrep-1_p20210321.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Grep-like tool to search for binary strings"
+HOMEPAGE="https://github.com/tmbinc/bgrep/"
+SRC_URI="https://github.com/tmbinc/bgrep/archive/master.tar.gz -> ${P}.tar.gz"
```
Comment (thesamesam) [inline app-misc/bgrep/bgrep-1_p20210321.ebuild:10]: This isn't pinned to a commit, so the hash is going to change on new commits. You want to fetch a particular commit from that day.

File: app-misc/bgrep/bgrep-1_p20210321.ebuild [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+KEYWORDS="~amd64"
+
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="test? ( dev-lang/perl )"
+
+src_prepare() {
+	default
+	sed -i -e "s|/tmp/|${T}/|g" \
```
Comment (thesamesam) [inline app-misc/bgrep/bgrep-1_p20210321.ebuild:26]: Nice.

**Review by thesamesam — CHANGES_REQUESTED** [review]

File: app-misc/bgrep/bgrep-1_p20210321.ebuild (hunk 1 above)
Comment (ScardracS) [reply app-misc/bgrep/bgrep-1_p20210321.ebuild]: Resolved

File: app-misc/bgrep/bgrep-1_p20210321.ebuild [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+EAPI=7
+
+GITHUB_USER="tmbinc"
+COMMIT="58c0b83f7ef13cd8d3d7352fadef88a006a514cd"
+
+inherit toolchain-funcs
+
+DESCRIPTION="Grep-like tool to search for binary strings"
+HOMEPAGE="https://github.com/tmbinc/bgrep/"
+SRC_URI="https://github.com/tmbinc/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
```
Comment (juippis) [inline app-misc/bgrep/bgrep-1_p20210321.ebuild:13]: Also please get in to habit of not typing `${PN}` here. In this case even it doesn't save ANY space from the string.
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D

**Review by juippis — APPROVED** [review]

File: app-misc/bgrep/Manifest [hunk 4]
```diff
@@ -1 +1,2 @@
+DIST bgrep-0.2_p20210321.tar.gz 4854 BLAKE2B 85c9171572e440f1e5def3a902a4adb1a3339706847013c6ff139745a819d9eab48a9449bf89b631cfcfe0081365240eefcae46a68064f710178db6d2db50037 SHA512 a9488ee9fd68a1b0e9924cbdf1e36fb3517cdceee4b57a27ef148ccd2c3f51942f8a57b41d47a2cb5b65b376b0b87c92ca4ee5d7053714a725c57ffbabbf12b6
```
Comment (juippis) [inline app-misc/bgrep/Manifest:1]: forgot to say, that we version the snapshot by the date of upstream commit, not by the date when ebuild was written. For obvious reason, helps to identify the commit when the snapshot was made, and prevents useless "version" bumps by writing the same ebuild on different date.

I fixed it while merging, but as a learning example for you.

Comment (scardracs) [reply app-misc/bgrep/Manifest:1]: Thanks :)

**Pull request closed without merging**